doc/explanation/components/components_overview.md

```diff
@@ -1,215 +1,298 @@
-# Component overview
+# Components Overview
```

Comment:
This is the guide originally contributed by @philippjfr as a tutorial. I don't believe its a tutorial. But its a very good explanation. And much better than the existing Components Overview.

Reply:
@maximlt wrote this one.

Reply:
😄 Its a really nice application. But especially the exceptions raised should be removed.

Reply:
I don't know if this is an explanation, at least I didn't write it with this intention. Maybe that's what it ended up being. Anyhow, I'd hope the tutorial would have some sort of explanation/teaching.